Makefile

```diff
@@ -0,0 +1,70 @@
+EXAMPLES := examples
+DASHBOARDS := ${EXAMPLES}/dashboards/
+ALERTS := ${EXAMPLES}/alerts.yaml
+RULES := ${EXAMPLES}/rules.yaml
+SRC = $(shell find . -type f -name '*.go' -not -path './vendor/*')
+JSONNET_SRC = $(shell find . -name 'vendor' -prune -o -name '*.libsonnet' -print -o -name '*.jsonnet' -print)
+
+JSONNET_FMT := jsonnetfmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
+
+CONTAINER_CMD:=docker run --rm \
+		-u="$(shell id -u):$(shell id -g)" \
+		-v "$(shell go env GOCACHE):/.cache/go-build" \
+		-v "$(PWD):/go/src/github.com/observatorium/thanos-receive-controller:Z" \
+		-w "/go/src/github.com/observatorium/thanos-receive-controller" \
+		-e USER=deadbeef \
+		-e GO111MODULE=on \
+		quay.io/coreos/jsonnet-ci
+
+all: generate fmt thanos-receive-controller
+
+thanos-receive-controller: main.go
+	go build -v
+
+.PHONY: generate
+generate: ${ALERTS} ${RULES} ${DASHBOARDS}
```

Comment:
Our CI should have a `generate` step that runs `make generate` and then ensures that `git diff --exit-code` exits clean.